scripts/run.bat

```diff
@@ -13,11 +13,11 @@ call .\.virtualenv\Scripts\activate.bat py35
 
 pushd condensation-forum
 
-echo Installing prerequisites
+echo installing prerequisites
```

Comment:
Fragments made lower case? This looks... wrong. They are supposed to be like capitalized headers. We could make all or at least the first word capitalized.

Reply:
one of the keys in vim is linked -> toLowerCase, and I hit it by accident sometimes.

Reply:
Ah! That makes much more sense.